### ISSUE TITLE
example project does not work

### ISSUE THREAD

**Issue by TaobotX11** [ArjanAswal/Stockfish#34]
hi, i try to run example in this project but i face this issue 

`Bad state: Stockfish is not ready (StockfishState.error)`

flutter doctor

```
[✓] Flutter (Channel stable, 3.10.2, on Microsoft Windows [Version 10.0.18363.592], locale en-US)
    • Flutter version 3.10.2 on channel stable at E:\Installers\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 9cd3d0d9ff (4 months ago), 2023-05-23 20:57:28 -0700
    • Engine revision 90fa3ae28f
    • Dart version 3.0.2
    • DevTools version 2.23.1
```

build gradle
```
android {
    compileSdkVersion 33
    ndkVersion "23.1.7779620"
}
```

```
dependencies {
        classpath 'com.android.tools.build:gradle:7.4.2'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
}
```
any workaround for this ?

thank you

**Comment by daohoangson** [comment]
```
The init isolate encountered an error Invalid argument(s): Failed to load dynamic library 'libstockfish.so': dlopen failed: library "libstockfish.so" not found
```

Apparently the app cannot find the library file. I'm investigating.

**Comment by daohoangson** [comment]
Did you test on a real device or emulator? Please include the device model / AVD configuration for further troubleshooting.

**Comment by eudorolshop** [comment]
> ```
> The init isolate encountered an error Invalid argument(s): Failed to load dynamic library 'libstockfish.so': dlopen failed: library "libstockfish.so" not found
> ```
> 
> Apparently the app cannot find the library file. I'm investigating.
> 
> Only happens in debug mode. It works in release build. Maybe related to #33...

i got same issue when i have no ndk installed, but after install ndk , this issue has gone.



> Did you test on a real device or emulator? Please include the device model / AVD configuration for further troubleshooting.

test on real device, 

`android-arm64  • Android 6.0.1 (API 23)`

thanks for comment

**Comment by daohoangson** [comment]
Yeah. It should work with NDK properly installed and devices in 3 architectures: ARM v8, ARM v7 and x86_64, no x86.

**Comment by ChopinDavid** [comment]
Can confirm, tried running on an x86 emulator and got the `Failed to load dynamic library 'libstockfish.so': dlopen failed: library "libstockfish.so" not found` error. Switching to an x86_64 emulator and was able to build.
